.test-defs/shootdns-test.yaml

```diff
@@ -0,0 +1,20 @@
+kind: TestDefinition
+metadata:
+  name: shootdns-test
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: System test for shoot DNS service
+
+  activeDeadlineSeconds: 7200
```

Comment:
Do you intentionally want to wait `2h` for a timeout from tm side? The test took so far between `5m` and `8m` in the last days, hence I suggest to put e.g. `30m` at max => `activeDeadlineSeconds: 1800`